### ISSUE TITLE
Test README example on CI

### ISSUE THREAD

**Issue by kvark** [kvark/froggy#15]
I believe it's currently broken, as of #13

**Comment by kvark** [comment]
One solution is to use `rust-skeptic`.
Another - is to add `rustdoc --test README.md` to the CI scrit. It's not clear if this is supposed to work, but it's currently blocked by inability to link `rustdoc` to the `Froggy` library.
Current state of the issue - no consensus, postponed till `rustdoc` is fixed.

**Comment by kvark** [comment]
The `cargo`/`rustdoc` doesn't appear to be in a rush to fix this - https://github.com/rust-lang/cargo/issues/383
Let's get the `skeptic` running then.
